### PR TITLE
Able to send announcement filecontent to hipchat

### PR DESCRIPTION
**Pull request by geiralund**
Let announcement file also be included in a hipcat notification.

### CODE REVIEW

**Comment by geiralund** [comment]
@michaelsembwever @eirslett Good / bad idea?

**Comment by eirslett** [comment]
Looks good to me!
But what about just publishing a release message to an infrastructure broker, and then forward the announcement from there on to the relevant channels? (E.g. when we're maybe moving to Slack, we don't have to rewrite the plugin)

File: pom.xml [hunk 1]
```diff
@@ -95,7 +95,7 @@
               <configuration>
                 <rules>
                   <requireJavaVersion>
-                    <version>[1.7.0-45,1.8)</version>
+                    <version>[1.7.0-45,1.8.0-31,1.8)</version>
```
Comment (michaelsembwever) [inline pom.xml]: wasn't really keen on allowing java8 builds of the project.
is this required in this pull request?

**Comment by michaelsembwever** [comment]
the idea is good.

File: pom.xml (hunk 1 above)
Comment (geiralund) [inline pom.xml]: No, not required at all. Will remove it.

**Comment by geiralund** [comment]
I really like the idea of publishing release messages on a broker @eirslett.  Should that be incorporated in this plugin (or a new one)? 

For now, i really like to see those messages be published asap.

**Comment by eirslett** [comment]
I think it fits inside this plugin.
Consider the cost of releasing the plugin twice (and rolling it out) vs the benefit of an immediate release. Either way should be fine, I guess.

**Comment by geiralund** [comment]
decided that publishing release messages to a broker belongs to another
pull request.

2015-03-02 10:00 GMT+01:00 Eirik Sletteberg notifications@github.com:

**Pull request merged**